realm/functions/get_token/source.js

```diff
@@ -0,0 +1,48 @@
+exports = async function(){
+  const GOOGLE_TOKEN_ENDPOINT = "https://oauth2.googleapis.com/token";
+
+  const CLIENT_ID = context.values.get("GOOGLE_CLIENT_ID");
+  const CLIENT_SECRET = context.values.get("GOOGLE_CLIENT_SECRET");
+  
+  const tokens_collection = context.services.get("mongodb-atlas").db("auth").collection("auth_tokens");
```

Comment:
Genuine question - not sure what the best practices are.  Are we comfortable hard-coding these strings? We could put them in values, but then to access them we still have to use a string.  Wish we had genuine constants we could reference, but I'm failing to think of a way we could do it.

Reply:
I know what you mean. The reason I assigned the collection to a variable was so that the database/collection was only hard-coded in once place.

If we put it in a value, it implies the database and collection will be different for different deployments. I suppose if you were connecting this Realm app to a cluster that's also used for other things, then you _might_ want to use a custom database/collection name?